presentation/frontend/src/test/favorite-properties/FavoritePropertiesStorage.test.ts

```diff
@@ -32,7 +33,9 @@ describe("IModelAppFavoritePropertiesStorage", () => {
     sinon.stub(IModelApp, "settings").get(() => settingsAdminMock.object);
 
     authorizationClientMock = moq.Mock.ofType<FrontendAuthorizationClient>();
-    authorizationClientMock.setup((x) => x.hasSignedIn).returns(() => true);
+    // eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
I don't understand what this comment is trying to disable. Is it necessary?

Reply:
Hmm, I think this was left over from a previous change I did and forgot to remove it. Thanks for pointing out.